RevenueCatUI/CustomerCenter/Views/ActiveSubscriptionsListView.swift

```diff
@@ -201,6 +249,13 @@ struct ActiveSubscriptionsListView: View {
             .disabled(self.viewModel.loadingPath != nil)
         }
     }
+
+    private var dateFormatter: DateFormatter {
+        let formatter = DateFormatter()
```

Comment:
I wonder if we should pass the chosen locale to the date formatter, to make sure it matches the locale used in the rest of CustomerCenter, in case the device locale is not supported...

Reply:
That's actually a great idea. 

We are not passing around the locale, but it's an easy tweak. In addittion, PurchaseInformation has an injected formatter, so I can use that one + locale to make sure it's correct 👍